guardian/prosemirror-elements
# feat: Add select all shortcut to TextField and RichTextField

## What does this change?

This adds a 'select all' shortcut to the TextField and RichTextField.

We filter the `baseKeymap` provided by `prosemirror-commands` here (based on key name) - removing the enter-based shortcuts which may interfere with intended behaviour in Composer.

| Before | After |
| ------  | ----- |
| ![2021-09-29 10 26 25](https://user-images.githubusercontent.com/34686302/135241921-0f1d9ece-3de4-4f9c-96e8-77df18f2c764.gif) | ![2021-09-29 10 25 53](https://user-images.githubusercontent.com/34686302/135241941-1302e28d-028f-43ab-8d54-2820861603eb.gif) |

## How to test

Run `yarn start` from the root directory locally (with nginx running) to run the application.

The new cypress tests should also pass - run them with `yarn test:integration` while the application is running in another terminal.

## Accessibility
